README.md

```diff
@@ -30,71 +30,88 @@ The Getting Started instructions can be found [here](GettingStarted.md).
   - [What If Two Arbitragers Compete For The Same Swap?](#what-if-two-arbitragers-compete-for-the-same-swap)
 - [Benchmarks and Fee Estimations](#benchmarks-and-fee-estimations-ymmv)
 - [Upgradability](#upgradability)
-- [Frontend Agnostic](#frontend-agnostic)
+- [Frontend Agnosticism](<#frontend agnosticism>)
 - [Conclusion](#conclusion)
 - [FAQ](#faq)
   - [Why Are Reference Scripts Required?](#if-all-users-have-direct-access-to-the-spending-script-for-that-trading-pair-why-are-reference-scripts-required)
 
 ---
+## Abstract
+`cardano-swaps` is a proof-of-concept implementation of massively scalable and concurrent DEX protocol for the Cardano Settlement Layer (CSL). It solves many of the pitfalls of current DEX implementations by empowering users to deploy their own (and interact with each others') script addresses. By doing so, users maintain fully custody of their assets at all times, and can elect if/when to upgrade their addresses to new contract standards.
```

Comment:
"massively" can be seen as hyperbolic. Instead, use "organically".

Reply:
You need to mention delegation control here. How about this (changes bolded):

By doing so, users maintain **full custody and delegation control** of their assets at all times, and can elect if/when to upgrade their addresses to new contract standards.

BTW, you said "fully custody"